### PR TITLE
Windows fix

### PR DESCRIPTION
**Pull request by berezh**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,17 +93,19 @@
     "babel-jest": "24.9.0",
     "chalk": "^2.4.1",
     "classnames": "^2.2.6",
+    "colors": "1.3.2",
```
Comment (oyershov) [inline package.json:96]: please, remove this package if you don't use it

**Comment by oyershov** [comment]
Great!🚀

File: package.json (hunk 1 above)
Comment (berezh) [reply package.json:96]: It's been used on mockserver.js file
`const colors = require('colors');`

Comment (oyershov) [reply package.json:96]: yes, but you don't use ```const colors```. Please remove it too

Comment (berezh) [reply package.json:96]: it's work on next way:
`
console.log('Reading from '+ mockFile.**yellow** +' file: ' + 'Matched 2'.**green**);
`

This file is coppied from `mockserver` package. But package is removed. All related libs, which been used inside are added.

**Pull request merged**